<launch>
	<!-- Start VRPN Client node: getting data from Optitrack -->
	<!-- Include  the file that contains your custom vrpn launch-->
	<include file="/home/marko/catkin_ws/src/vrpn_client_ros/launch/across.launch"/>

	<!-- Start Optitrack node: calculating and distributing postion data -->
	<node pkg="tic_tac_drone" type="OptitrackNode.py" name="Optitrack">
		<remap from="MyUAV/pose" to="vrpn_client_node/UAV1/pose"/>
		<remap from="OpUAV/pose" to="vrpn_client_node/UAV2/pose"/>
		<remap from="Field/pose" to="vrpn_client_node/Field/pose"/>
	</node>

</launch>